network-api/networkapi/buyersguide/templates/buyersguide/fragments/pni_nav_links.html

```diff
@@ -0,0 +1,4 @@
+{% load bg_nav_tags i18n wagtailroutablepage_tags %}
+
+{% routablepageurl page 'about-why-view' as about_why_url %}
+{{ pre }}<a class="{{ class }} {% bg_active_nav request.get_full_path about_why_url %}" href="{{ about_why_url }}">{% trans "About the Guide" %}</a>{{ post }}
```

Comment:
Use a routable page view link as to not link to the original /privacynotincluded/ urls in the Django app